api/src/main/java/jakarta/persistence/NamedEntityGraph.java

```diff
@@ -71,6 +71,13 @@
  * }
  * }
  *
+ * <p> The root entity of the named graph is identified by it being the
+ * unique entity class that is not referenced by the {@code subgraph}
+ * memnber of any {@link NamedEntityGraphSubgraph} annotation belonging
+ * to the named graph. If there is no such entity class, or it is not
+ * unique, the behavior is undefined. An application must ensure that
+ * each named entity graph has a unique root entity.
```

Comment:
I have added this very important clarification to the semantics of `@NamedEntityGraphSubgraph`. Without it, the whole design doesn't make sense.